components/sup/src/ctl_gateway/mod.rs

```diff
@@ -22,37 +22,32 @@
 
 pub mod server;
 
-use std::{borrow::Cow,
-          fmt,
+use std::{fmt,
           fs::{self,
                File},
           io::{self,
                Write},
           path::Path};
 
-use regex::Regex;
-use termcolor::{Buffer,
-                Color,
+use termcolor::{Color,
```

Comment:
It would be good to remove the newline that separates `std` from everything else for consistency, and to let `rustfmt` handle everything for us.